jest.preset.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+const path = require('path');
```

Comment:
Curious what the reasoning is for having this at the root rather than under `scripts` where we've generally been putting shared config? (In retrospect I should potentially have asked the same about `tsconfig.base.json` when it was checked in.)